src/EditorFeatures/Core/Shared/Tagging/TagProviders/AbstractAsynchronousBufferTaggerProvider.cs

```diff
@@ -1,74 +0,0 @@
-// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
-
-using System;
-using Microsoft.CodeAnalysis.Editor.Shared.Tagging.TagSources;
-using Microsoft.CodeAnalysis.Editor.Tagging;
-using Microsoft.CodeAnalysis.Shared.TestHooks;
-using Microsoft.VisualStudio.Text;
-using Microsoft.VisualStudio.Text.Editor;
-using Microsoft.VisualStudio.Text.Tagging;
-
-namespace Microsoft.CodeAnalysis.Editor.Shared.Tagging
```

Comment:
this file was renamed to AsynchronousBufferTaggerProvider.cs.  Git doesn't seem to have recognized that.